tests/k256_ecdh.cc

```diff
@@ -154,15 +153,15 @@ TEST_P(K256EcdhWycheproof, KAT)
   WycheproofEcdh test = GetParam();
 
   // Obtain public key from test.
-  vector<uint64_t> public_key(15);
+  bytes public_key(64);
   {
+    // Instead of parsing ASN.1, we just ignore the prefix ...
+    bytes prefix = from_hex("3056301006072a8648ce3d020106052b8104000a034200");
+    bytes pk = bytes(test.public_key_asn1.begin() + prefix.size(), test.public_key_asn1.end());
     if (test.CompressedPoint) {
-      cout << "Skipping. Compressed points are no longer supported." << endl;
-      return;
+      Hacl_K256_ECDSA_public_key_compressed_to_raw(public_key.data(), pk.data());
     } else {
-      // TODO(https://github.com/cryspen/hacl-packages/issues/157)
-      cout << "Skipping. Only compressed points are supported for now." << endl;
-      return;
+      Hacl_K256_ECDSA_public_key_uncompressed_to_raw(public_key.data(), pk.data());
```

Comment:
This segfaults for tcId 199 because your `pk` is empty.
The HACL APIs aren't defensive, so you have to do these checks here.